core/core/sustainability_labels/sustainability-labels-evaluation.csv

```diff
@@ -4,7 +4,7 @@ certificate:BIORE,bioRe,Fashion,0,0,0,0,0,0,0,0,0,0,0,0,0,0
 certificate:BLUE_ANGEL,Blauer Engel,Fashion,93,86,0,81,100,100,0,24,40,55,0,0,0,0
 certificate:BLUE_ANGEL_LAPTOPS,Blauer Engel - Laptops,Electronics,73,77,87,0,0,100,40,71,0,0,0,0,0,0
 certificate:BLUE_ANGEL_LEATHER,Blaue Engel - Leder,Fashion,73,67,0,100,60,0,0,0,0,22,0,0,0,0
-certificate:BLUE_ANGEL_SMARTPHONE,Blaue Engel - Smartphones,Electronics/Smartphones,73,71,83,0,0,50,25,67,0,72,0,0,0,100
+certificate:BLUE_ANGEL_SMARTPHONES,Blaue Engel - Smartphones,Electronics/Smartphones,73,71,83,0,0,50,25,67,0,72,0,0,0,100

```

Comment:
We do not have this label (`BLUE_ANGEL_SMARTPHONE`) in the DB yet as it was not covered by the mapping in `_certificate_category_names `yet, right?

Reply:
Right, we don't have it in the `green-db` table at all but we do have it in the `sustainability-labels` table.
But even under this name `BLUE_ANGEL_SMARTPHONE` it doesn't have any scores so we might as well do a manual delete together in the future? But otherwise it won't affect anything from now on as we also removed the singular SMARTPHONE in `_certificate_category_names`

Reply:
Ok, great 👍 I think with the new date for the sustainability labels in `bootstrap_database` we won't have the old version in the DB anymore. So I think we are fine here :)